packages/server/test/support/fixtures/cloud/studio/test-studio.ts

```diff
@@ -0,0 +1,8 @@
+import type { AppStudioShape } from '@packages/types'
+import type { Router } from 'express'
+
+export class AppStudio implements AppStudioShape {
+  initializeRoutes (router: Router): void {
+
```

Comment:
is this implementation meant to be empty?

Reply:
Yeah this is just a fixture and we're not really testing anything specific with it currently.